aws/resource_aws_chime_voice_connector_termination.go

```diff
@@ -0,0 +1,191 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"regexp"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/chime"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceAwsChimeVoiceConnectorTermination() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceAwsChimeVoiceConnectorTerminationPut,
+		ReadContext:   resourceAwsChimeVoiceConnectorTerminationRead,
+		UpdateContext: resourceAwsChimeVoiceConnectorTerminationUpdate,
+		DeleteContext: resourceAwsChimeVoiceConnectorTerminationDelete,
+
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"calling_regions": {
+				Type:     schema.TypeList,
+				Required: true,
+				MinItems: 1,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.StringLenBetween(2, 2),
+				},
+			},
+			"cidr_allow_list": {
+				Type:     schema.TypeList,
+				Required: true,
+				MinItems: 1,
+				Elem: &schema.Schema{
+					Type:         schema.TypeString,
+					ValidateFunc: validation.IsCIDRNetwork(27, 32),
```

Comment:
just want to double check this one: do the docs reference the min netmask?

Reply:
what range would you recommend? there are different ranges used thought-out the codebase